libs/gstsavantframemeta/gstsavantframemeta/src/savantnvprobes.cpp

```diff
@@ -0,0 +1,55 @@
+#include "savantnvprobes.h"
+#include <vector>
+#include "gstnvdsmeta.h"
+
+
+GstPadProbeReturn remove_tracker_objs_pad_probe(GstPad *pad,
+                                                GstPadProbeInfo *info,
+                                                gpointer user_data) {
+    GstBuffer *buffer = GST_PAD_PROBE_INFO_BUFFER(info);
+    if (!buffer) {
+        GST_INFO_OBJECT(pad, "Tracker obj remover: skipping NULL buffer.");
```

Comment:
The log entry contains filename, line and name of method. There's no need to add a prefix to the log lines.

Reply:
Removed the logging prefix.